examples/vertex_mlops_enterprise/doc/01-ENVIRONMENTS.md

```diff
@@ -45,8 +45,8 @@ Setup your new Github repo using the Github web console or CLI.
 Copy the `vertex_mlops_enterprise` folder to your local folder, including the Github actions:
 
 ```
-cp -R ./examples/vertex_mlops_enterprise/* ./<YOUR LOCAL FOLDER>
-cp -R ./examples/vertex_mlops_enterprise/.github ./<YOUR LOCAL FOLDER>
+# Copy base directory from git repo, including hidden dirs and files
```

Comment:
@javiergp it's a bit redundant with the line 45

Reply:
fixed